lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -307,8 +307,7 @@ module InternalConstants
 
     # Whether valid entries should be written even if some other entries fail
     # due to INVALID_ARGUMENT or PERMISSION_DENIED errors when communicating to
-    # the Stackdriver Logging API. This is highly recommended. Right now this
-    # only works with the REST path (use_grpc = false).
+    # the Stackdriver Logging API. This is highly recommended.
     config_param :partial_success, :bool, :default => false
```

Comment:
Would it make sense to also flip the default to `true`? This shouldn't break existing customers, right?

Reply:
Good idea. Changed.